manifests/m/Microsoft/AppInstaller/1.26.400.0/Microsoft.AppInstaller.installer.yaml

```diff
@@ -0,0 +1,39 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Microsoft.AppInstaller
+PackageVersion: 1.26.400.0
+Platform:
+- Windows.Universal
+MinimumOSVersion: 10.0.17763.0
+InstallerType: msix
+Commands:
+- winget
+FileExtensions:
+- winget
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCLibs.Desktop.14
+    MinimumVersion: 14.0.33728.0
+  - PackageIdentifier: Microsoft.UI.Xaml.2.8
+    MinimumVersion: 8.2310.30001.0
+PackageFamilyName: Microsoft.DesktopAppInstaller_8wekyb3d8bbwe
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/microsoft/winget-cli/releases/download/v1.11.400/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle
+  InstallerSha256: 7F626B4206BC4546086A0A70C901A66CA09D653AE01412A68C1FD9351386AD3B
+  SignatureSha256: 3B4669425587B5DEAF175825490051898F94DF608E3FED24A35BE3012EAAA97F
+- Architecture: x64
+  InstallerUrl: https://github.com/microsoft/winget-cli/releases/download/v1.11.400/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle
+  InstallerSha256: 7F626B4206BC4546086A0A70C901A66CA09D653AE01412A68C1FD9351386AD3B
+  SignatureSha256: 3B4669425587B5DEAF175825490051898F94DF608E3FED24A35BE3012EAAA97F
+- Architecture: arm
+  InstallerUrl: https://github.com/microsoft/winget-cli/releases/download/v1.11.400/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle
+  InstallerSha256: 7F626B4206BC4546086A0A70C901A66CA09D653AE01412A68C1FD9351386AD3B
+  SignatureSha256: 3B4669425587B5DEAF175825490051898F94DF608E3FED24A35BE3012EAAA97F
```

Comment:
```suggestion
```

`arm` no longer supported in this one

Reply:
Right. And I'm realizing that I didn't update the hash either... 😅